about.html

```diff
@@ -0,0 +1,123 @@
+                    </ul>
+                </div>
+                <div class="desk-nav">
+                    <div class="logolink">
+                      <a class="logolink1" href="#"><span class="desklogo">VR Odyssey</span></a>
+                    </div>
+                    <div class="deskuldiv">
+                    <ul class="deskul2">
+                      <li class="desklist2"><a href="./index.html" class="desklink1">home</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">program</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Join</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">Sponser</a></li>
+                      <li class="desklist2"><a href="#" class="desklink1">News</a></li>
+                    </ul>
+                    <div class=desbutton><button class="deskbutton">VR Campaign</button></div>
+                </div>
+                </div>
+                <div class="home">
+                    <ul class="homeul">
+                      <li class="homeli"><a href="./index.html" class="desklink2"><i class="fa-solid fa-house homeicon"></a></i></li>
+                      <li class="homeli"><a href="#" class="desklink2"><i class="fa-solid fa-angle-right"></a></i></li>
+                      <li class="homeli"><a href="./about.html" class="desklink2">about</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <section class="page">
+            <p class="para pick">"Welcome to VR!"</p>
+            <div class="clip">
+                <h1 class="heading1 pick">VR EVOLUTION:<br>THE NEXT CHAPTER 2023</h1>
+            </div>
+            <hr class="aboutline pick">
+            <div class="paraborder">
+                <p class="para1 pick">We are excited to bring together professionals from across the VR industry for this two-day event exploring the latest trends, innovations, and challenges in VR technology. From keynote speeches and panel discussions to workshops, demonstrations, and networking events, we have designed an engaging and informative program that will provide attendees with valuable insights and practical skills.</p>
+            </div>
+            <div class="venue">
+                <p class="time pick">Please contact us per Email for any further questions<br>about VR EVO. THE NEXT CHAPTER 2023</p>
+                <p class="place pick">vrodyssey2023@evo.org</p>
+            </div>
+        </section>
+    </section>
+        <section class="challenge">
+            <div class="heading2">
+               <h2 class="heading3 pick">VR Escape Room Challenge</h2>
+               <hr class="line pick">
+            </div>
+            <p class="propara pick">A VR escape room challenge where attendees work in teams to solve puzzles and escape a virtual environment within a set time frame.</p>
```

Comment:
- [OPTIONAL] Kindly consider decreasing the size and weight of this paragraph and applying a lighter color like `#e2e2e2` to make it similar to the Original design.